### PR TITLE
Fix BulkRestApiSpec

### PR DESCRIPTION
**Pull request by snimavat**
connect #348

### CODE REVIEW

File: examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-import spock.lang.Ignore
 import spock.lang.Specification
+import yakworks.rally.job.Job
 import yakworks.rally.orgs.model.Org
 
 @Rollback
 @Integration
-
 class BulkRestApiSpec extends Specification implements OkHttpRestTrait, JsonParserTrait {
     String path = "/api/rally/org/bulk?jobSource=Oracle"
```
Comment (basejump) [inline examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy:17]: jobSource should not be passed in, only jobSourceId.
we need to test that job.source get populated with the url (controller/action? is what got mocked in) that called it

File: examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
-    //XXX https://github.com/yakworks/gorm-tools/issues/348 can we test Job here?
     void "verify bulk create and sanity check response"() {
```
Comment (basejump) [inline examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy:19]: test jobSource id and that the job.source got populated with the url that called it

File: examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy (hunk 1 above)
Comment (jdabal) [reply examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy:17]: @Josh forgot to tell you today that I changed source with sourceid:
sourceId -- url controller/action
source - param jobSource, like Oracle -- /api/ar/customer/bulk?jobSource=Oracle

File: examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy [hunk 3]
```diff
@@ -44,6 +43,21 @@ class BulkRestApiSpec extends Specification implements OkHttpRestTrait, JsonPars
...
+        job.data != null
+        job.state == JobState.Finished
+
+        when: "Verify job.data json, this is what come in from the request"
+        List dataList = parseJson(job.requestData)
+
+        then:
+        dataList.size() == 3
+        dataList[0].num == "foox1"
+
```
Comment (jdabal) [inline examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy:60]: @snimavat can we test source and sourceid here ?

Comment (snimavat) [reply examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy:64]: Done

File: examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy (hunk 1 above)
Comment (snimavat) [reply examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy:17]: Test updated

File: examples/restify/src/integration-test/groovy/restify/OrgRestApiSpec.groovy [hunk 4]
```diff
@@ -243,34 +243,4 @@ class OrgRestApiSpec extends Specification implements OkHttpRestTrait {
...
-
-        Map body = bodyToMap(resp)
-
-        then:
-        resp.code() == HttpStatus.CREATED.value()
-        //def job = Job.get(resp.id)
-        //job.sourceId == "/api/rally/org/bulkCreate"
-        //job.source == "Oracle"
-        body.source == "Oracle"
-
```
Comment (snimavat) [inline examples/restify/src/integration-test/groovy/restify/OrgRestApiSpec.groovy:273]: This detailed asserts are now in unit tests BulkableRepoSpec - Basic sanity check in BulkableRestApiSpec

**Pull request merged**